vm/runtime.ts

```diff
@@ -2097,8 +2224,21 @@ var AOTMD = J2ME.aotMetaData;
  * Are we currently unwinding the stack because of a Yield? This technically
  * belonges to a context but we store it in the global object because it is
  * read very often.
+ *
+ * We define it as a getter/setter to enable breaking when it is set in Chrome:
+ *   http://stackoverflow.com/questions/11618278/how-to-break-on-property-change-in-chrome
  */
-var U: J2ME.VMState = J2ME.VMState.Running;
+var _U: J2ME.VMState = J2ME.VMState.Running;
```

Comment:
> This checked extremely often, can we remove this or enable it only for debugging?

Indeed, and it isn't likely to be used very often (I only did so once, when I initially added it to troubleshoot a bug), so I've reverted it in ba95470 (leaving this version in a comment in case someone needs it in the future).